### PR TITLE
Add configuration for Data Center URL

### PR DESCRIPTION
**Pull request by mincledon**
Add configuration snippet need to set URL per Data Center

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,15 @@ EventsApi api = new ApiClient("API KEY", "API-SECRET")
                 .createService(EventsApi.class);
 ```
 
+By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you can add the following configuration to the SDK initialization:
+
+```java
+Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("DATA CENTER URL").build();
```
Comment (samdozor) [inline README.md:174]:
```suggestion
Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("https://s2s.eu1.mparticle.com/v2/").build();
```

Comment (samdozor) [reply README.md:174]: if i were a developer i'd want to know if i need to start it with https, end it with a slash, should it contain `/events` etc - so better to include a real example

File: README.md [hunk 2]
```diff
@@ -168,6 +168,15 @@ EventsApi api = new ApiClient("API KEY", "API-SECRET")
...
+EventsApi api = new ApiClient("API KEY", "API-SECRET")
+                .setAdapterBuilder(mybuilder)
+                .createService(EventsApi.class);
+```
+
```
Comment (samdozor) [inline README.md:179]: [See the mParticle docs site](https://docs.mparticle.com/developers/data-localization/#events-api) to find the right URL for your hosting location.

File: README.md [hunk 3]
```diff
@@ -168,6 +168,15 @@ EventsApi api = new ApiClient("API KEY", "API-SECRET")
                 .createService(EventsApi.class);
 ```
 
+By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you can add the following configuration to the SDK initialization:
```
Comment (samdozor) [inline README.md:171]:
```suggestion
By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you must override the base url using the following configuration:
```

**Review by samdozor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mincledon) [reply README.md:174]: smart!

File: README.md (hunk 2 above)
Comment (mincledon) [reply README.md:179]: @samdozor I added the piece about defaulting to US1 Data Center, which links to mP docs.  What are you suggesting with this comment?

**Pull request merged**